Dockerfile

```diff
@@ -0,0 +1,28 @@
+# CodeClimate specification: https://github.com/codeclimate/spec/blob/master/SPEC.md
+#
+# Build and run via:
+#   docker build -t codeclimate/codeclimate-reek . && docker run codeclimate/codeclimate-reek
+
+FROM codeclimate/alpine-ruby:b38
```

Comment:
Hmm, b38 seems to be five months old, but still the newest version. Does Code Climate has a plan to maintain this image?